HEN_HOUSE/scripts/egsnrc_bashrc_additions

```diff
@@ -89,7 +89,7 @@ alias pprocess='$HEN_HOUSE/scripts/pprocess'
 # hard-coded run-time search path (no -rpath)
 #
 if test -d $HEN_HOUSE/egs++/dso/linux-static; then
-   alias egs_view='(LD_LIBRARY_PATH=${HEN_HOUSE}egs++/dso/linux-static/ ${HEN_HOUSE}pieces/linux/egs_view_64)'
+   alias egs_view='LD_LIBRARY_PATH=${HEN_HOUSE}egs++/dso/linux-static/ ${HEN_HOUSE}pieces/linux/egs_view_64'
```

Comment:
**Wow!** I did not know about this single-exectuable level of scoping for shell variables in bash! Leaving out the semi-colon after the variable definition implies the variable is strictly only available for the executable immediately following. Oh well, you learn some (bash) thing everyday. So there are **three**, not two, levels of scoping on the command-line; noting them here for reference:
```
key=value a.out    # key only defined for executable a.out (try: key=value env | grep key; env | grep key)
key=value          # key only defined in current shell
export key         # key defined in current shell and its sub-processes (through the environment)
```